indicators/indicators.py

```diff
@@ -197,6 +194,143 @@ def run_compute_indicators(fp_di, indicators, coord_labels, kwargs={}):
     return out
 
 
+def build_attrs(
+    indicator, scenario, model, start_year, end_year, lat_min, lat_max, lon_min, lon_max
+):
+    """Build standardized attribute dictionarys for computed indicator datasets. This function uses lookup tables imported from indicators/luts.py.
+
+    Args:
+        indicator (str): indicator id
+        model (str): model id
+        scenario (str): scenario id
+        start_year (str): first year of dataset
+        end_year (str): last year of dataset
+        lat_min (str): minimum latitude of dataset
+        lat_max (str): maximum latitude of dataset
+        lon_min (str): minimum longitude of dataset
+        lon_max (str): maximum longitude of dataset
+
+    Returns:
+        global_attrs, var_coord_attrs (tuple): tuple of global and variable/coordinate attribute dictionarys
+    """
+    # test units to determine NA value TODO: revisit this if there are any other integer units besides "days"
+    if units_lu[indicator] != "d":
+        fill_value = "NaN"
+    else:
+        fill_value = "-9999"
+
+    # build global attribute dict for the whole dataset:
+    global_attrs = {
+        "title": f"{indicator_lu[indicator]['title']}, {start_year}-{end_year}: {model}-{scenario}",
+        "author": "Scenarios Network for Alaska and Arctic Planning (SNAP), International Arctic Research Center, University of Alaska Fairbanks",
+        "creation_date": datetime.datetime.now().strftime("%m/%d/%Y, %H:%M:%S"),
+        "email": "uaf-snap-data-tools@alaska.edu",
+        "website": "https://uaf-snap.org/",
+        "references": f"{indicator_lu[indicator]['references']}",  # TODO: add reference information!
```

Comment:
See comments below about model and scenario attributes. I would say those attributes could go here, since they really apply to all data in the file. e.g.
```suggestion
        "references": f"{indicator_lu[indicator]['references']}",  # TODO: add reference information!
        "scenario": f"{scenario}",
        "model": f"{model}",
        "institution": f"{model_lu[model]['institution']}",
        "institution_name": f"{model_lu[model]['institution_name']}",
```

Reply:
Agreed, seems more appropriate to have this info in global attributes instead of coordinate attributes.